### DISCUSSION TITLE
No text found! View "Whisper_log.txt"?

### DISCUSSION THREAD

**Discussion by khanhtoan8135** [SubtitleEdit/subtitleedit#7718] [General]
### **I am experiencing an issue and need help. When using SubtitleEdit software to create Japanese subtitles, I encounter an error as shown in the image below. The English subtitles, however, are working fine. I appreciate any assistance, thank you very much.**

![Screenshot 2023-12-08 122736](https://github.com/SubtitleEdit/subtitleedit/assets/99197814/59562b1e-21a6-434e-b7b2-fa7ab5ea1968)

![Screenshot 2023-12-08 122625](https://github.com/SubtitleEdit/subtitleedit/assets/99197814/7cc2e547-ef27-4024-bd06-121838ebcd2e)

**Comment by Purfview** [comment]
You have "out of memory error", if English worked then try to reduce VRAM usage a bit, the first setting to try - `--best_of=1`.

Btw, I've seen people complain that on Japanese audio `large-v3` is worse than `large-v2`.
